server/controllers/index.js

```diff
@@ -97,8 +98,12 @@ router.get('/api/reset-cookie', (req, res) => {
 router.post('/api/playCardWord', auth, (req, res) => {
     if (gameLogic.isCurrentPlayer(req.session.user)) { /* Only current player is allowed to play both a word and a card */
         try {
-            gameLogic.playCardAndWord(req.session.user, req.body.cardId, req.body.word)
-            res.sendStatus(200);
+            if (validWord.isValidWord(req.body.word)) {
```

Comment:
Could possibly move this check into gameLogic and have it throw an error as we are doing in other places?